x-pack/plugins/security_solution/public/flyout/shared/components/flyout_body.tsx

```diff
@@ -18,7 +19,16 @@ interface FlyoutBodyProps extends React.ComponentProps<typeof EuiFlyoutBody> {
  */
 export const FlyoutBody: FC<FlyoutBodyProps> = memo(({ children, ...flyoutBodyProps }) => {
   return (
-    <EuiFlyoutBody {...flyoutBodyProps}>
+    <EuiFlyoutBody
+      {...flyoutBodyProps}
+      css={css`
+        .euiFlyoutBody__overflow {
+          // fix a bug with red overlay when position was not set
+          // remove when position: relative is added in EUI
+          mask-image: none;
```

Comment:
instead of removing the mask entirely, you can just do `transform: translateZ(0);` and the mask should work, and won't break once eui is upgraded.

Reply:
done, thanks!